src/app/GitExtUtils/GitUI/ControlDpiExtensions.cs

```diff
@@ -102,6 +103,31 @@ public static void AdjustForDpiScaling(this Control control)
                         upDown.Margin = DpiUtil.Scale(new Padding(3));
                         break;
                     }
+
+                    case ToolStrip toolStrip:
+                    {
+                        foreach (ToolStripItem toolStripItem in toolStrip.FindToolStripItems())
+                        {
+                            switch (toolStripItem)
+                            {
+                                case ToolStripComboBox toolStripComboBox:
```

Comment:
less `break`s, please:
```suggestion
                            if (toolStripItem is ToolStripComboBox toolStripComboBox)
```

Reply:
With this and `ToolStripUtil.FindToolStripItems` I was following an already present code structure which probably made more sense in those cases as they were handling multiple types. I think that if there is already a piece of code that solves a particular problem with switch+case then if you want to do the same just on different types it is IMO a better readability UX if the same switch+case pattern is followed as that cognitive "price" of understanding it has already been paid.

I'll follow your suggestion, just wanted to explain my reasoning.